plugin/src/test/java/io/jenkins/plugins/casc/impl/configurators/DataBoundConfiguratorTest.java

```diff
@@ -1,15 +1,13 @@
 package io.jenkins.plugins.casc.impl.configurators;
 
-import io.jenkins.plugins.casc.ConfigurationAsCode;
-import io.jenkins.plugins.casc.ConfigurationContext;
-import io.jenkins.plugins.casc.Configurator;
-import io.jenkins.plugins.casc.ConfiguratorRegistry;
+import io.jenkins.plugins.casc.*;
```

Comment:
Please configure your imports according to the project standard

A-Z imports, no star, static imports last.

Reply:
Yeah forgot the import opitmization :P. I will include it in my next commit.Thanks a lot.